scripts/visualize_signals.py

```
@@ -0,0 +1,294 @@
+        """Visualize API metrics if available"""
+        if not self.check_api_health():
+            print("❌ API not available at", self.api_url)
+            print("💡 Start the API server first:")
+            print("   docker-compose up galactus-core")
+            return
+
+        print("🔍 Fetching API metrics...")
+
+        try:
+            # Get health status
+            health_response = requests.get(f"{self.api_url}/api/v1/health")
```

Ruff (0.14.10)

Probable use of `requests` call without timeout

(S113)